addons/bitbucket/models.py

```diff
@@ -193,6 +193,11 @@ def is_private(self):
         connection = BitbucketClient(access_token=self.api.fetch_access_token())
         return connection.repo(user=self.user, repo=self.repo)['is_private']
 
+    @property
+    def fetched_repo(self):
```

Comment:
Sorry to backtrack on a previous review, but it's unintuitive for a property to do an HTTP request. Attribute access shouldn't have expensive side effects. How about making this a getter like `.fetch_repo()` (as a method, not a property)?

Note: I see that `is_private` is making an HTTP request, but that's a bad example.